src/cli.rs

```diff
@@ -153,6 +154,12 @@ pub struct Opt {
     /// one into the other.
     #[structopt(long = "max-line-distance", default_value = "0.3")]
     pub max_line_distance: f64,
+
+    /// Whether to use a pager when displaying output. Options are: auto, always, and never. The
+    /// default pager is `less`: this can be altered by setting the environment variables BAT_PAGER
+    /// or PAGER (BAT_PAGER has priority).
+    #[structopt(long = "paging", default_value = "auto")]
+    pub paging_mode: String,
```

Comment:
Since `delta` is in some ways similar to [bat](https://github.com/sharkdp/bat), I've made it a design guideline to be exactly the same as bat wherever it makes sense. bat has basically exactly the option that you're adding, so I've pushed commits to your PR making the interface identical. Here's a section from `bat --help` for reference.

![image](https://user-images.githubusercontent.com/52205/75630479-bbb82280-5bb0-11ea-8bcb-49520c8de3e7.png)

Reply:
Sorry I should be totally explicit here, since lazy git is going to pass this option: I have changed the name of the option in your PR so that it will now be `--paging=never`.